msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -21,37 +18,55 @@ The process to update your search-based Teams message extension to run Outlook i
+1. From Visual Studio Code, open the command palette (`Ctrl+Shift+P`), type `Teams: Create a new Teams app`
+1. Select **Create a new Teams app** option
+1. Select **Search-based message extension** to download sample code for a Teams message extension using the latest Teams app manifest (version `1.13`)
 
-:::image type="content" source="images/toolkit-search-sample.png" alt-text="NPM Search Connector sample in Teams Toolkit":::
+    :::image type="content" source="images/toolkit-palatte-search-sample.png" alt-text="Type 'Create a new Teams app' VS Code command palette to list Teams sample options":::
+
+    The sample is also available as *NPM Search Connector* in the Teams Toolkit Samples gallery. From the Teams Toolkit pane, select *Development* > *View samples* > **NPM Search Connector**.
+    
+    :::image type="content" source="images/toolkit-search-sample.png" alt-text="NPM Search Connector sample in Teams Toolkit Samples gallery":::
+
+1. Select a location on your local machine for the workspace folder
+1. Open the command palette (`Ctrl+Shift+P`) and type `Teams: Provision in the cloud` to create the required app resources (Azure App Service, App Service plan, Azure Bot, and Managed Identity) in your Azure account.
+1. Open the command palette (`Ctrl+Shift+P`) and type `Teams: Deploy to the cloud` to deploy the sample code to the provisioned resources in Azure and start the app.
+
+From here, you can skip ahead to [Add an Outlook channel for your bot](#add-an-outlook-channel-for-your-bot) to complete the final step of enabling the Teams message extension to work in Outlook. (The app manifest is already referencing the correct version, so no updates are necessary.)
 
 ## Update the app manifest
 
-You'll need to use the [Teams developer preview manifest](/microsoftteams/platform/resources/schema/manifest-schema-dev-preview) schema and the `m365DevPreview` manifest version to enable your Teams message extension to run in Outlook.
+You'll need to use the [Teams developer manifest](../resources/schema/manifest-schema.md) schema version `1.13` to enable your Teams message extension to run in Outlook.
 
 You have two options for updating your app manifest:
 
 # [Teams Toolkit](#tab/manifest-teams-toolkit)
 
-1. Open the *Command palette*: `Ctrl+Shift+P`
+1. Open the command palette: `Ctrl+Shift+P`
 1. Run the `Teams: Upgrade Teams manifest to support Outlook and Office apps` command and select your app manifest file. Changes will be made in place.
 
```

Comment:
Saurabh Bhatia suggested us to change some strings for these two upgrade commands to remove the Outlook and Office part because you can continue to target Teams as well. So we updated the commands as bellow:

* `Teams: Upgrade Teams JS SDK and code references`
* `Teams: Upgrade Teams manifest`

Reply:
Updated